crates/node/core/build.rs

```diff
@@ -1,19 +1,20 @@
 #![allow(missing_docs)]
 
 use std::{env, error::Error};
-use vergen::EmitBuilder;
+use vergen::{Emitter, BuildBuilder, CargoBuilder};
+use vergen_git2::Git2Builder;
 
 fn main() -> Result<(), Box<dyn Error>> {
     // Emit the instructions
-    EmitBuilder::builder()
-        .git_describe(false, true, None)
-        .git_dirty(true)
-        .git_sha(false)
-        .build_timestamp()
-        .cargo_features()
-        .cargo_target_triple()
-        .emit_and_set()?;
-
+    
+    let mut emitter = Emitter::default();
+    emitter.add_instructions(&BuildBuilder::default().build()?)?;
```

Comment:
```suggestion
    emitter.add_instructions(&BuildBuilder::default().build_timestamp(true).build()?)?;
```
the default value here is false, check out source code https://docs.rs/vergen/9.0.4/src/vergen/feature/build.rs.html#152

so we need to explicitly enable it with available method, check out 
https://docs.rs/vergen/9.0.4/vergen/struct.BuildBuilder.html#method.build_timestamp